minitest-snapshots.gemspec

```diff
@@ -9,15 +9,11 @@ Gem::Specification.new do |spec|
   spec.authors = ["Harry Brundage"]
   spec.email = ["harry.brundage@gmail.com"]
 
-  spec.summary = %q{Adds jest style snapshot testing asserting against long values stored on the filesystem.}
+  spec.summary = %q{Minitest plugin implementing Jest-style snapshot testing}
```

Comment:
Whoopth

Reply:
Not sure what this means :-) Should I revert this?

Reply:
No no just saying thanks for fixing it !